nixos/modules/system/boot/systemd.nix

```diff
@@ -829,16 +829,9 @@ in
 
     system.build.units = cfg.units;
 
-    # Systemd provides various NSS modules to look up dynamic users, locally
-    # configured IP adresses and local container hostnames.
-    # On NixOS, these can only be passed to the NSS system via nscd (and its
-    # LD_LIBRARY_PATH), which is why it's usually a very good idea to have nscd
-    # enabled (also see the config.nscd.enable description).
-    # While there is already an assertion in place complaining loudly about
-    # having nssModules configured and nscd disabled, for some reason we still
-    # check for nscd being enabled before adding to nssModules.
-    system.nssModules = optional config.services.nscd.enable systemd.out;
-    system.nssDatabases = mkIf config.services.nscd.enable {
+    # enable the systemd-provided nss modules
```

Comment:
this states the obvious, what make you think this comment is good? Maybe we can improve on it if it is important. We went from a very large (somtimes vague) comment to a very obvious statement.

Reply:
The comment in `nsswitch.nix` is essentially what explains why we need to pass nss modules via nscd, and why it's a bad idea to not do it. I could just drop this comment here, as it just writes what's written again in code immediately afterwards.

Reply:
I dropped the comment.